pkg/placement/plugins/tainttoleration/taint_toleration.go

```diff
@@ -91,7 +91,7 @@ func (pl *TaintToleration) Filter(ctx context.Context, placement *clusterapiv1be
 func (pl *TaintToleration) RequeueAfter(ctx context.Context, placement *clusterapiv1beta1.Placement) (plugins.PluginRequeueResult, *framework.Status) {
 	status := framework.NewStatus(pl.Name(), framework.Success, "")
 	// get exist decisions clusters
-	decisionClusterNames, decisionClusters := getDecisionClusters(pl.handle, placement)
+	decisionClusterNames, decisionClusters := getDecisionClusters(klog.FromContext(ctx), pl.handle, placement)
```

Comment:
Same as above, this can bring benefits to all the plugins.

Reply:
updated it